### DISCUSSION TITLE
blog/headless-ui-vs-component-library

### DISCUSSION THREAD

**Discussion by giscus** [hendraaagil/website#44] [Comments]
# blog/headless-ui-vs-component-library

Kapan kita harus memilih untuk menggunakan Headless UI daripada Component Library?

https://hendraaagil.dev/blog/headless-ui-vs-component-library

**Comment by faiisll** [comment]
Berasa banget pas butuh komponen mulai kompleks, karna terlanjur pake component library akhirnya di mix pake library lain2 lagi, jadi berasa berantakan project nya.

**Reply by hendraaagil** [reply]
refactor solusinya 😅

**Reply by faiisll** [reply]
rada susah mas, karna project masih nambah fitur terus, blm dikasih waktu buat bisa refactor 😅

**Reply by hendraaagil** [reply]
hahaha, takdir emg berkata lain 😂
